extension/src/popup/components/sendPayment/SendSettings/Settings/index.tsx

```diff
@@ -190,52 +190,50 @@ export const Settings = ({
           <Form className="View__contentAndFooterWrapper">
             <View.Content>
               <FormRows>
-                {!isToken ? (
```

Comment:
I haven't been able to track down the original reason for doing this but this seems to work as is when removing this constraint.